motor/core.py

```diff
@@ -486,7 +495,11 @@ async def coro(session, custom_arg, custom_kwarg=None):
                 return ret
 
     def start_transaction(
-        self, read_concern=None, write_concern=None, read_preference=None, max_commit_time_ms=None
+        self,
+        read_concern=None,
+        write_concern=None,
+        read_preference=None,
+        max_commit_time_ms=None,
```

Comment:
What's up with these changes?

Reply:
I think that's the auto-formatter at work. I let it be as I assumed it came from the pre-commit hook.

Reply:
Yeah, this is a byproduct of running `tox -m lint` before I pushed.

Reply:
I've gone ahead and re-added it, but as a separate commit.